test/test.dart

```diff
@@ -532,26 +532,31 @@ extension on Map<String, String?> {
 
 BaasClient? _baasClient;
 Future<void> setupBaas() async {
-  final baasUrl = arguments[argBaasUrl];
-  if (baasUrl == null) {
-    return;
+  if (BaasClient.initializationError != null) return;
```

Comment:
why do we need this as static on the `BaasClient`? Can we move it to module level variable?

Reply:
Done.